regulations/tasks.py

```diff
@@ -72,10 +74,16 @@ def submit_comment(self, body, files):
     except (ClientError, RequestException) as exc:
         logger.exception(exc)
         raise self.retry(exc=exc)
+    except Task.MaxRetriesExceededError as exc:
```

Comment:
Is this block reachable? It looks like `MaxRetriesExceededError` would be thrown from inside the previous `except`--will that be caught here, or do we need a nested `try`?